package/kedro_viz/models/flowchart.py

```diff
@@ -9,6 +9,7 @@
 from types import FunctionType
 from typing import Any, ClassVar, Dict, List, Optional, Set, Union, cast
 
+from fastapi.encoders import jsonable_encoder
```

Comment:
I wonder in future, if we can replace this with basic encoders. This is for removing reliance on FastAPI dependency when embedded Kedro-viz.